### PR TITLE
D2B : add new IPF for the latest cycle

### PR DESCRIPTION
**Pull request by gvardany**
**Description of work.**

This commits new IPF and IDF for D2B valid from 2018 May 1st containing a recent tube alignment measurement from the latest cycle.

**Report to:** ILL/Cuello. 

**To test:**

Check the valid-from dates to match. Parameters are validated by the scientist.

No issue.

Does this update require release notes?
- [ ] Yes
- [x] No

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: instrument/D2B_Definition_2018-05-01.xml [hunk 1]
```diff
@@ -0,0 +1,325 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- For help on the notation used to specify an Instrument Definition File see http://www.mantidproject.org/IDF -->
+<instrument xmlns="http://www.mantidproject.org/IDF/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 Schema/IDFSchema.xsd" name="D2B" valid-from="2018-05-01 23:59:59"
+valid-to="2100-01-31 23:59:59" last-modified="2018-03-20 14:05:02">
```
Comment (KarlPalmen) [inline instrument/D2B_Definition_2018-05-01.xml]: Do you really mean `valid-from="2018-05-01 23:59:59`?

File: instrument/D2B_Parameters_2018-05-01.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<parameter-file instrument="D2B" valid-from="2018-05-01 23:59:59">
+	<component-link name="D2B">
```
Comment (KarlPalmen) [inline instrument/D2B_Parameters_2018-05-01.xml]: Matches valid from date in IDF

**Review by KarlPalmen — CHANGES_REQUESTED** [review]
The dates are consistent, but suspicious to me. Do you want the files to be valid from one second before the start of 2 May?

**Comment by gvardany** [comment]
The cycle started mid May, and the previous one ended 26 April, so it just has to be something in between.

**Review by KarlPalmen — APPROVED** [review]

**Pull request merged**